api/src/functions/hello/index.ts

```diff
@@ -0,0 +1,20 @@
+import schema from './schema'
+import {handlerPath} from '@lib/handlerResolver'
+import type {FunctionDefinition} from '@lib/serverless'
+
+export const hello: FunctionDefinition = {
```

Comment:
We use our own declaration here because otherwise this is just a blind object and offers no key hints.